src/wp-admin/import.php

```diff
@@ -190,11 +218,12 @@
 	}
 	?>
 </table>
-<?php
+	<?php
 }
 
-if ( current_user_can('install_plugins') )
-	echo '<p>' . sprintf( __('If the importer you need is not listed, <a href="%s">search the plugin directory</a> to see if an importer is available.'), esc_url( network_admin_url( 'plugin-install.php?tab=search&type=tag&s=importer' ) ) ) . '</p>';
+if ( current_user_can( 'install_plugins' ) ) {
+	echo '<p>' . sprintf( __( 'If the importer you need is not listed, <a href="%s">search the plugin directory</a> to see if an importer is available.' ), esc_url( network_admin_url( 'plugin-install.php?tab=search&type=tag&s=importer' ) ) ) . '</p>';
```

Comment:
Should this `sprintf` be expanded?

Reply:
This makes sense if we start to observe `translator` comments, will add to PR update.

Reply:
PR updated